oracle/src/lib.rs

```diff
@@ -1,30 +1,78 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use support::{decl_event, decl_module, decl_storage};
+mod mock;
+mod operator_protocol;
+mod tests;
+mod timestamped_value;
+
+pub use operator_protocol::OperatorProtocol;
+
+use rstd::prelude::Vec;
+use rstd::*;
```

Comment:
```suggestion
use rstd::prelude::*;
```

and remove line above